source/routing/specifying-a-routes-model.md

```diff
@@ -204,7 +202,31 @@ will always execute the model hook.
 If your data represented by your model is being updated frequently, you may
 want to refresh it periodically:
 
-<a class="jsbin-embed" href="http://jsbin.com/sefuv/2/embed?js">JS Bin</a><script src="http://static.jsbin.com/js/embed.js"></script>
+```app/routes/pull-requests.js
+export default Ember.Route.extend({
+  model() {
+    var url = 'https://api.github.com/repos/emberjs/ember.js/pulls';
+    return Ember.$.getJSON(url).then(function(data) {
```

Comment:
what do you have mind @locks? Also the point of this PR is just to get rid of the jsbin - I think changing around the examples on this page is out of scope, although may warrant a new issue.